homeassistant/components/ecobee/climate.py

```diff
@@ -300,6 +300,17 @@ def __init__(self, data, thermostat_index):
         self.vacation = None
         self._last_active_hvac_mode = HVAC_MODE_HEAT_COOL
 
+        # Check that this is a known model, log message if not
+        if not self.thermostat["modelNumber"] in ECOBEE_MODEL_TO_NAME:
+            _LOGGER.error(
```

Comment:
We don't want side effect like logging in init methods.

Reply:
I've moved this out of the init method. It is now in a Throttled method called by `async_update`. The current setting logs errors once an hour to avoid too much log polution.